app.py

```diff
@@ -391,45 +396,45 @@ def run_optimization(
 
```

Comment:
Won't any attempt to solve a problem that size fail at the solver step?

Reply:
Just tested (was unsure if it actually would work) and the solver actually finished for both k-means and NL with problems that are large enough to cause overflow errors for the line above. This would be a simple fix, but we could also support larger numbers by calculating it in a smarter way, but that's probably a bit overkill here.

Reply:
There seemed to be a dash-issue when storing/loading the callback outputs though, causing it to sometimes get stuck after the run is complete. Not sure about what's going on there. 🫤